test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/TemplateTest.cs

```diff
@@ -43,14 +43,30 @@ public void TestPredictProgram_WithSampleData()
         [Fact]
         [UseReporter(typeof(DiffReporter))]
         [MethodImpl(MethodImplOptions.NoInlining)]
-        public void TestConsumeModel()
+        public void TestConsumeModel_NonAzureImage()
         {
             var consumeModel = new ConsumeModel()
             {
                 Namespace = "Namespace",
-                HasNormalizeMapping = true,
-                HasLabelMapping = true,
-                MLNetModelpath = @"/path/to/model",
+                IsAzureImage = false,
+                MLNetModelName = @"mlmodel.zip",
+            };
+
+            Approvals.Verify(consumeModel.TransformText());
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void TestConsumeModel_AzureImage()
+        {
+            var consumeModel = new ConsumeModel()
+            {
+                Namespace = "Namespace",
+                IsAzureImage = true,
+                MLNetModelName = @"mlmodel.zip",
+                OnnxLabelName = "onnx.json",
+                OnnxModelName = "onnx.onnx"
```

Comment:
As per repo norm, can you add an issue for this PR and a PR description?

via the [contribution guide](https://github.com/dotnet/machinelearning/blob/master/CONTRIBUTING.md#pull-requests):
> Your pull request needs to reference a filed issue. Please fill in the template that is populated for the pull request. Only pull requests addressing small typos can have no issues associated with them.

Reply:
Added
### Resolved